op-node/rollup/driver/l1_state.go

```diff
@@ -20,6 +20,10 @@ type L1State struct {
 	l1Head      eth.L1BlockRef
 	l1Safe      eth.L1BlockRef
 	l1Finalized eth.L1BlockRef
+
+	haveL1Head      bool
+	haveL1Safe      bool
+	haveL1Finalized bool
```

Comment:
I still like to use the zeroed values for the have checks, instead of introducing these booleans in the state. The boolean in the return parameters is only there to force us to not assume the value isn't zero. We can check that in the getters too. If we introduce separate state for that, then it might create more edge cases where the value is zeroed but the haveL1X is true.